project_recalculate/README.rst

```diff
@@ -0,0 +1,83 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Project recalculate
+===================
+
+This module allows to recalculate tasks' start/end dates depending on project's
+start/end dates.
+
+There are two calcutation modes:
+
+* **Date begin**: Task's start/end dates are recalculated from project's date begin
+* **Date end**: Task's start/end dates are recalculated from project's date end
+
+If the project start/end date is changed in the form view, then you have to
+click "Recalculate proyect" button to recalculate all pending tasks
+according to the new date.
+
+
```

Comment:
Add an installation section, mentioning that an Odoo or OCB version from 29-08-2015 or later is required, because the odoo#8208 fix is needed to work properly.